pkg/utils/utils.go

```diff
@@ -0,0 +1,157 @@
+package utils
+
+import (
+  "math/rand"
+  "time"
+  "io/ioutil"
+  "github.com/sirupsen/logrus"
+
+  "k8s.io/api/core/v1"
+  apps "k8s.io/api/apps/v1"
+  metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+  route "github.com/openshift/api/route/v1"
+)
+
+func GetFileContents(filePath string) []byte {
+  contents, err := ioutil.ReadFile(filePath)
+  if err != nil {
```

Comment:
Do we need to return the err or is it sufficient to return a nil contents?

Reply:
We could do that, I'll be honest that I didn't focus much on error handling for this first pass

Reply:
Going to defer a solution on this until after all component PRs have merged in